nrf_rpc/include/nrf_rpc.h

```diff
@@ -32,8 +32,18 @@ extern "C" {
 /** @brief Special value to indicate that ID is unknown or irrelevant. */
 #define NRF_RPC_ID_UNKNOWN 0xFF
 
+/** @brief Flag indicating that the group does not block on initialization. */
+#define NRF_RPC_FLAGS_WAIT_ON_INIT BIT(0)
+
+/** @brief Flag indicating that the peer must initiate group binding. */
+#define NRF_RPC_FLAGS_INITIATOR BIT(1)
+
+/** @brief Helper macro for conditional flag initialization. */
+#define NRF_RPC_FLAG_COND(cond, flag) (cond ? flag : 0UL)
```

Comment:
To be safe when using this macro pleas add brackets
```suggestion
#define NRF_RPC_FLAG_COND(_cond, _flag) ((_cond) ? (_flag) : 0UL)
```

Reply:
Done